src/routes/mountains.js

```diff
@@ -0,0 +1,28 @@
+export default function Mountains() {
```

Comment:
when using the app open developer console, toggle device tool bar ctrl+shift+m and play with the different devices there. You will see that the mountains sometimes blocks the login form. We should fix this.